etl/transformer/transformer.go

```diff
@@ -173,7 +173,8 @@ func initRecordsMapsByObj(records []types.Record) (
 }
 
 func restoreInsolarID(b []byte) string {
-	if bytes.Equal(b, []byte{}) {
+	emptyByte := make([]byte, len(b))
+	if bytes.Equal(b, []byte{}) || bytes.Contains(b, emptyByte) {
```

Comment:
Why  `bytes.Contains` ?

Reply:
during the test, we can initialise the record ref, PrototypeReference and etc, with length but filled zeroing. 
that is not correct value and Mainnet can call panic

Reply:
Why `Contains` and not `Equal` than ?

Reply:
my mistake. tank you